src/components/Courses/components/EmptyCourseList/EmptyCourseList.tsx

```diff
@@ -1,31 +1,40 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 
 import { Button } from 'src/common/Button';
-import { TOKEN_KEY_NAME } from 'src/util/CommonConstant';
+import {
+	PERMISSION_ERROR_TEXT_MESSAGE,
+	TOKEN_KEY_NAME,
+} from 'src/util/CommonConstant';
+import { RouterPath } from 'src/util/RouterPath';
 
 import './EmptyCourseList.css';
 
+const EMPTY_COURSE_LIST_TITLE = 'Course List is Empty';
+const EMPTY_COURSE_LIST_MESSAGE =
+	'Please use "Add New Course" button to add your first course';
+const ADD_NEW_COURSE_BUTTON_TEXT = 'ADD NEW COURSE';
+
 export const EmptyCourseList: React.FC = () => {
-	if (localStorage.getItem(TOKEN_KEY_NAME) === null) {
-		return (
-			<div className='emptyCourseList'>
-				<h1 id='titleId'>Course List is Empty</h1>
-				<text>
-					You don't have permissions to create a course. Please login as ADMIN
-				</text>
-			</div>
-		);
-	} else {
-		return (
-			<div className='emptyCourseList'>
-				<h1 id='titleId'>Course List is Empty</h1>
-				<text>Please use "Add New Course" button to add your first course</text>
-				<Button
-					text='Add New Course'
-					onClick={() => console.log('You clicked')}
-					id={''}
-				/>
-			</div>
-		);
-	}
+	const navigate = useNavigate();
+
+	const addCourseOnClick = () => {
+		//add role check in module 4
+		if (
+			localStorage.getItem(TOKEN_KEY_NAME) === null ||
```

Comment:
you can check here as: 
```
!localStorage.getItem(TOKEN_KEY_NAME)
```
because it will return true is item is null or it is an empty string